easybuild/easyconfigs/b/Beast/Beast-10.5.0-GCC-13.3.0.eb

```diff
@@ -0,0 +1,35 @@
+easyblock = 'Tarball'
+
+name = 'Beast'
```

Comment:
this should be renamed to BEAST X, as it is a different implementation than BEAST 2